service/models/wishlist.py

```diff
@@ -109,11 +109,6 @@ def init_db(cls, app:Flask):
     logger.info("Wishlist: Initializing database")
     cls.app = app
```

Comment:
Do we even need this method at all? I don't see `cls.app` or `self.app` being referenced anywhere in the class. Or am I missing something on how this is used?

Reply:
Good point. I'm actually not sure about this. But, if you refer to the code at line 25, ```app``` is initialized to be ```None```. I think what I have here is to assign an app instance to this field. Depending on we are doing testing or receiving requests from user, ```app``` contains different config information which might be useful when Flask-SQLAlchemy performs query?